### PR TITLE
docs: Remove network dependency in readme_test

### PR DESCRIPTION
**Pull request by kklin**
The README has code that resolves GitHub keys.

### CODE REVIEW

File: docs/readme_test.go [hunk 1]
```diff
@@ -112,6 +115,12 @@ func TestReadme(t *testing.T) {
 }
 
 func checkConfig(content string, quiltPath string) error {
+	stitch.HTTPGet = func(url string) (*http.Response, error) {
+		resp := http.Response{
+			Body: ioutil.NopCloser(bytes.NewBufferString("")),
+		}
+		return &resp, nil
+	}
```
Comment (mmussomele) [inline docs/readme_test.go]: restore `stitch.HTTPGet` after?

**Comment by mmussomele** [comment]
Other than the one comment, looks good.

File: docs/readme_test.go (hunk 1 above)
Comment (kklin) [inline docs/readme_test.go]: Eh, tests run in isolation, I think it'd just make the test slightly harder to follow.

Comment (kklin) [inline docs/readme_test.go]: Actually, Matt swayed me otherwise.

**Pull request merged**